app/components/listing/ServiceAttribution.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ServiceVerified from '../../assets/img/ic-verified-record.svg';
+
+import './ServiceAttribution.scss';
+
+class ServiceAttribution extends React.Component {
+  isVerified() {
+    const { attribution, status } = this.props;
+    return attribution === 'service_net' && status === 'approved';
+  }
+
+  render() {
+    const benetechLink = 'https://medium.com/@Shelter_Tech/sheltertech-is-participating-'
+      + 'in-the-benetech-service-net-pilot-in-the-san-francisco-bay-area-b28645d3dee6';
+
+    return this.isVerified() ? (
```

Comment:
Is this the right name for the method? We really want to include ByServiceNet in the name or something similar right? There's a different field is_verified which we use sounds ambiguous to this method.

Reply:
yes, that is a very good point. i think this was leftover confusion as to whether i was working on HAP or ServiceNet or both.  i have updated the method name as well as some markup class names and img name for greater clarity.